benchmark/fixtures/simple-https-server.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const fixtures = require('../../test/common/fixtures');
+const https = require('https');
+
+const options = {
+  key: fixtures.readKey('rsa_private.pem'),
+  cert: fixtures.readKey('rsa_cert.crt')
+};
+
+const storedBytes = Object.create(null);
+const storedBuffer = Object.create(null);
+
+module.exports = https.createServer(options, (req, res) => {
+  // URL format: /<type>/<length>/<chunks>/chunkedEnc
+  const params = req.url.split('/');
+  const command = params[1];
+  let body = '';
+  const arg = params[2];
+  const n_chunks = parseInt(params[3], 10);
+  const chunkedEnc = params.length >= 5 && params[4] === '0' ? false : true;
+  let status = 200;
+
+  let n, i;
+  if (command === 'bytes') {
+    n = ~~arg;
+    if (n <= 0)
+      throw new Error('bytes called with n <= 0');
+    if (storedBytes[n] === undefined) {
+      storedBytes[n] = 'C'.repeat(n);
+    }
+    body = storedBytes[n];
+  } else if (command === 'buffer') {
+    n = ~~arg;
+    if (n <= 0)
+      throw new Error('buffer called with n <= 0');
+    if (storedBuffer[n] === undefined) {
+      storedBuffer[n] = Buffer.allocUnsafe(n);
+      for (i = 0; i < n; i++) {
+        storedBuffer[n][i] = 'C'.charCodeAt(0);
+      }
+    }
+    body = storedBuffer[n];
+  } else {
+    status = 404;
+    body = 'not found\n';
+  }
+
+  // example: http://localhost:port/bytes/512/4
```

Comment:
```suggestion
  // example: https://localhost:port/bytes/512/4
```

Reply:
Good catch. Fixed in 6675aa02fc19340967a093c1f8108a0772acafb7